src/checkHoliday.js

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env node
+
+const Holidays = require('date-holidays')
+const country = 'CA';
+const hd = new Holidays(country);
+const path = require("path");
+const fs = require("fs");
+const holidayDatesPath = path.join(__dirname, '../config-files/holidays.txt');
+const nationalHolidays = hd.getHolidays(2019);
+
+var employeeHolidays = [];
+
+//console.log(nationalHolidays);
+
+nthWeekdayOfMonth = (weekday, n, date) => {
+    var count = 0,
+        idate = new Date(date.getFullYear(), date.getMonth(), 1);
+    while (true) {
+      if (idate.getDay() === weekday) {
+        if (++count == n) {
+          break;
+        }
+      }
+      idate.setDate(idate.getDate() + 1);
+    }
+    return idate;
+}
+
+writeHolidays = () => {
+    nationalHolidays.forEach(holiday => {
+        if (holiday.type === 'public' && new Date(holiday.date).getMonth() != 0 && new Date(holiday.date).getMonth() != 11 && holiday.name != 'Remembrance Day' && holiday.name != 'Easter Sunday') {
```

Comment:
No need to create two Date objects. Create it once and use it in both checks. 

Use strict inequality (`!==`) whenever possible.